packages/components/src/components/Search/Search.tsx

```diff
@@ -1,35 +1,118 @@
-import React from "react";
-import { View } from "react-native";
-import { EDSStyleSheet } from "../../styling";
-import { Input, TextFieldProps, useStyles } from "../..";
 import { MaterialIcons } from "@expo/vector-icons";
+import React, { useEffect, useRef, useState } from "react";
+import { Animated, View } from "react-native";
+import { TextInput } from "react-native-gesture-handler";
+import { Button, Input, TextFieldProps, useStyles } from "../..";
+import { EDSStyleSheet } from "../../styling";
 
-export type SearchProps = Omit<TextFieldProps, "multiline">;
-
-export const Search = (props: SearchProps) => {
+export type SearchProps = Omit<TextFieldProps, "multiline"> & {
+    allowCancel?: boolean;
```

Comment:
Should we perhaps rename this to `cancellable` something more clean? What do you think ?